src/Symfony/Component/ErrorHandler/ErrorEnhancer/ClassNotFoundErrorEnhancer.php

```diff
@@ -155,6 +155,14 @@ private function convertFileToClass(string $path, string $file, string $prefix):
             }
         }
 
+        // Symfony may ship some polyfills, like "Normalizer". But if the Intl
+        // extension is already installed, the next require_once will fail with
+        // a compile error because the class is already defined. And this one
+        // does not throw a Throwable. So it's better to skip it here.
+        if (str_contains($file, 'Resources/stubs')) {
```

Comment:
To me, this looks like the root cause of your issue is that the candidate is not properly identified to skip this on line 153